### PR TITLE
DEV2-3284: Add Velero resource.

### PR DESCRIPTION
**Pull request by SimSimY**
This PR adds an option to create `Velero` backup schedule as part of the helm installation. It is disabled by default.

### CODE REVIEW

File: charts/tabnine-cloud/templates/backup.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{{- if .Values.backup.enabled -}}
+apiVersion: velero.io/v1
+kind: Schedule
+metadata:
+  name: {{ .Release.Namespace }}-{{ .Release.Name }}
+  namespace: {{ .Values.backup.namespace }}
+spec: # Schedule is a Cron expression defining when to run the Backup
```
Comment (assapir) [inline charts/tabnine-cloud/templates/backup.yaml:7]: I prefer to have this comment on the `values.yaml` and regenerate the readme

File: charts/tabnine-cloud/values.yaml [hunk 2]
```diff
@@ -106,10 +106,9 @@ auth:
     host: null
     # -- Ignore TLS errors when connecting to the smtp server
     ignoreSmtpTls: false
-    auth:
-      null
-      # user: null
-      # pass: null
+    auth: null
```
Comment (assapir) [inline charts/tabnine-cloud/values.yaml:109]: seems like a mistake?

**Review by assapir — APPROVED** [review]
just regenrate the readme

File: charts/tabnine-cloud/templates/backup.yaml (hunk 1 above)
Comment (SimSimY) [reply charts/tabnine-cloud/templates/backup.yaml:7]: There is documentation in the values.

File: charts/tabnine-cloud/values.yaml (hunk 2 above)
Comment (SimSimY) [reply charts/tabnine-cloud/values.yaml:109]: was "broken" (chack masters version)

**Pull request merged**